sdk/samples/iot/docs/how_to_iot_hub_samples_linux.md

```diff
@@ -12,7 +12,7 @@ What is covered:
 - Downloading and building the Azure SDK for Embedded C suite
 - Configuring and running the IoT Hub client samples.
 
-    _The following was run on an Ubuntu Desktop 18.04 environment, but it also works on WSL 1 and 2 (Windows Subsystem for Linux)_
+_The following was run on an Ubuntu Desktop 18.04 environment, but it also works on WSL 1 and 2 (Windows Subsystem for Linux)._
```

Comment:
Remove the `_` likely the linter will not like this.

Reply:
Linter was fine with it :)